start_index: 0
waypoints:
   robot_type: [ 0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,    0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,    0.0,    0.0,   0.0,    0.0,    0.0,    0.0,   0.0,   0.0,   0.0,   0.0,   0.0,    0.0,    0.0,    0.0,    0.0,    0.0,    0.0,    0.0,    0.0,    0.0,    0.0,    0.0,    0.0,     0.0,    0.0,    0.0,     0.0,    0.0]
   robot_id: [   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,    1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,    1.0,    1.0,   1.0,    1.0,    1.0,    1.0,   1.0,   2.0,   2.0,   2.0,   2.0,    2.0,    2.0,    2.0,    2.0,    2.0,    2.0,    2.0,    2.0,    2.0,    2.0,    2.0,    2.0,     2.0,    2.0,    2.0,     2.0,    2.0]
   x:        [ 15.00, 10.00, 16.00, 23.00, 28.00, 34.00, 32.00, 30.00, 30.00, 10.00, 26.00,  26.00, 20.00, 15.00,  0.00, 15.00, 13.20, 21.00,  7.00, 16.00, 32.00, 14.00,  0.00,   0.0,  7.00, 22.00, 36.00, 13.00,   0.0, 10.00, 20.00, 35.00, 39.00, 40.00,  41.00,  40.00, 32.00,  21.00,   6.00,  -3.00,  0.00,  8.00, 10.00,-20.00,-24.00, -37.00, -41.00, -42.00, -30.00, -12.00, -29.00, -40.00, -33.00, -19.00, -17.00, -12.00, -24.00,  -26.00, -39.00, -39.00,  -25.00, -12.00]
   y:        [ -4.00,-30.80,-47.00,-41.00,-35.00,-26.00, -8.00, 10.00, 18.50, -2.00,  2.00, -13.00,-30.00, -8.00,-12.00, -4.00, 28.41, 37.00, 46.00, 46.00, 32.00, 12.00,-12.00,-30.00,-45.00,-60.00,-45.00,-30.00,-12.00, -5.00, 12.00, 23.00,  7.00, -7.00, -24.00, -43.00,-60.00, -65.00, -55.00, -35.00,-12.00, 15.00, 20.00, 20.00, 34.00,  57.00,  40.00,  26.00,   6.00, -12.00, -26.00, -44.00, -49.00, -55.00, -36.00, -12.00,  16.00,   17.00,   0.00, -10.00,  -11.00, -12.00]
   type:     [     2,     2,     2,     2,     2,     2,     2,     2,     2,     2,     2,      2,     2,     2,     4,     2,     2,     2,     2,     2,     2,     2,     4,     2,     2,     2,     2,     2,     4,     2,     2,     2,     2,     2,      2,      2,     2,      2,      2,      2,     2,     2,     2,     2,     2,      2,      2,      2,      2,      2,      2,      2,      2,      2,      2,      4,      2,      2,       2,      2,       2,      4]

   # x: [ 7.00, 37.65, 22.24, 18.00, 10.00, 37.30, 47.50, 2.00, 48.88, 43.35, 2.00, 37.46, 24.27, 2.00, 7.00, -30.00, -80.00, -80.00, -10.00, -80.00, -75.00, -10.00, -70.00, -60.00, -10.00, -50.00, -35.00, -10.00]
   # y: [ 7.00, -13.00, -24.01, -39.00, 2.00, -2.00, 0.80, 2.00,  12.00, 17.14, 1.00, 25.60, 39.08, 1.00, 7.00, 23.00 ,0.00, 20.00, -2.00, 10.00, 30.00, -2.00, 40.00, 60.00, -2.00, 65.00, 75.00, -2.00]
